src/main/java/com/github/jreddit/parser/listing/RedditListingParser.java

```diff
@@ -0,0 +1,150 @@
+package com.github.jreddit.parser.listing;
+
+import static com.github.jreddit.parser.util.JsonUtils.safeJsonToString;
+
+import java.util.LinkedList;
+import java.util.List;
+
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import org.json.simple.parser.ParseException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.github.jreddit.parser.entity.Comment;
+import com.github.jreddit.parser.entity.Kind;
+import com.github.jreddit.parser.entity.Submission;
+import com.github.jreddit.parser.entity.Subreddit;
+import com.github.jreddit.parser.entity.Thing;
+import com.github.jreddit.parser.util.JsonUtils;
+import com.github.jreddit.request.error.RedditError;
+
+public class RedditListingParser {
+	
+	/** Logger for this class. */
+	final static Logger LOGGER = LoggerFactory.getLogger(SubmissionsListingParser.class);
+	
+	/** JSON parser. */
+	protected JSONParser jsonParser;
+	
+	/**
+	 * Constructor.
+	 */
+	public RedditListingParser() {
+		jsonParser = new JSONParser();
```

Comment:
I think you can make the declaration of the jsonParser as a static final instead in the constructor.